apps/client/src/views/backstage/Backstage.tsx

```diff
@@ -0,0 +1,178 @@
+import Empty from '../../common/components/state/Empty';
+import TitleCard from '../../common/components/title-card/TitleCard';
+import ViewLogo from '../../common/components/view-logo/ViewLogo';
+import ViewParamsEditor from '../../common/components/view-params-editor/ViewParamsEditor';
+import { useWindowTitle } from '../../common/hooks/useWindowTitle';
+import { ViewExtendedTimer } from '../../common/models/TimeManager.type';
+import { cx, timerPlaceholderMin } from '../../common/utils/styleUtils';
+import { formatTime, getDefaultFormat } from '../../common/utils/time';
+import SuperscriptTime from '../../features/viewers/common/superscript-time/SuperscriptTime';
+import { useTranslation } from '../../translation/TranslationProvider';
+import ScheduleExport from '../common/schedule/ScheduleExport';
+
+import { getBackstageOptions, useBackstageOptions } from './backstage.options';
+import { getCardData, getIsPendingStart, getShowProgressBar, isOvertime } from './backstage.utils';
+
+import './Backstage.scss';
+
+interface BackstageProps {
+  backstageEvents: OntimeEvent[];
+  customFields: CustomFields;
+  eventNext: OntimeEvent | null;
+  eventNow: OntimeEvent | null;
+  general: ProjectData;
+  isMirrored: boolean;
+  time: ViewExtendedTimer;
+  runtime: Runtime;
+  selectedId: string | null;
+  settings: Settings | undefined;
+}
+
+export default function Backstage(props: BackstageProps) {
+  const {
+    backstageEvents,
+    customFields,
+    eventNext,
+    eventNow,
+    general,
+    time,
+    isMirrored,
+    runtime,
+    selectedId,
+    settings,
+  } = props;
+
+  const { getLocalizedString } = useTranslation();
+  const { secondarySource } = useBackstageOptions();
+  const [blinkClass, setBlinkClass] = useState(false);
+  const { height: screenHeight } = useViewportSize();
+
+  useWindowTitle('Backstage');
+
+  // blink on change
+  useEffect(() => {
+    setBlinkClass(false);
+
+    const timer = setTimeout(() => {
+      setBlinkClass(true);
+    }, 10);
+
+    return () => clearTimeout(timer);
+  }, [selectedId]);
+
+  // gather card data
+  const hasEvents = backstageEvents.length > 0;
+  const { showNow, nowMain, nowSecondary, showNext, nextMain, nextSecondary } = getCardData(
+    eventNow,
+    eventNext,
+    'title',
+    secondarySource,
+    time.playback,
+  );
+
+  // gather timer data
+  const clock = formatTime(time.clock);
+  const isPendingStart = getIsPendingStart(time.playback, time.phase);
+  const startedAt = isPendingStart ? formatTime(time.secondaryTimer) : formatTime(time.startedAt);
+  const scheduledStart =
+    hasEvents && showNow ? '' : formatTime(runtime.plannedStart, { format12: 'hh:mm a', format24: 'HH:mm' });
+  const scheduledEnd =
+    hasEvents && showNow ? '' : formatTime(runtime.plannedEnd, { format12: 'hh:mm a', format24: 'HH:mm' });
+
+  let stageTimer = millisToString(time.current, { fallback: timerPlaceholderMin });
+  stageTimer = removeLeadingZero(stageTimer);
```

Comment:
We call this stage timer but don't follow the same commends that the stage timer dose eg show message/aux/external

might be ok...

Reply:
thats a good point

I think the showrunner may want the presenter to see different things now and then.
However, the crew backstage is mostly interested in how long its left for the next thing

However, this does not solve the case where the showrunner wants the people backstage to see an aux timer... Maybe we can take that another time once we have a clear use case?

Reply:
sounds good, but maybe just call it timer and not stage timer?

Reply:
sounds good, called it `displayTimer`